src/util/str.rs

```diff
@@ -112,7 +112,7 @@ pub trait AsciiCast {
 
 impl AsciiCast for char {
     fn to_ascii_opt(&self) -> Option<Ascii> {
-        let n = *self as uint;
+        let n = *self as usize;
```

Comment:
This should be `u32` I guess, because it comes from a `char`.  I'm not sure why it was `uint`, but I think I copied this code from libstd.